pandas/core/series.py

```diff
@@ -2576,10 +2576,136 @@ def _needs_reindex_multi(self, axes, method, level):
         """
         return False
 
-    @Appender(generic._shared_docs['align'] % _shared_doc_kwargs)
     def align(self, other, join='outer', axis=None, level=None, copy=True,
               fill_value=None, method=None, limit=None, fill_axis=0,
               broadcast_axis=None):
+        """
+        Align two Series on their axes with the
+        specified join method for each axis Index.
+
+        The result is a tuple containing two Series with the same indexes.
+        Using `join`, one can specifiy if the union or intersection of the
+        indexes are retained or whether to keep the index of the first or
+        second Series.
+
+        Align supports MultiIndexing, `level` can be used to select which index
+        that is used for alignment.
+
+        Parameters
+        ----------
+        other : DataFrame or Series
+            The Series object to align with.
+        join : {'outer', 'inner', 'left', 'right'}, default 'outer'
+            How to aggregate the indexes.
+        axis : allowed axis of the other object, default None
+            Align on index (0), columns (1), or both (None).
+        level : int or level name, default None
+            Broadcast across a level, matching Index values on the
+            passed MultiIndex level.
+        copy : boolean, default True
+            Always returns new objects. If copy=False and no reindexing is
```

Comment:
I wuld write `False` instead of `copy=False`

Reply:
ok